TECHMANIA/Assets/Scripts/Locale.cs

```diff
@@ -165,6 +165,10 @@ public static string GetString(string key)
         params object[] args)
     {
         string format = GetString(formatKey);
-        return string.Format(format, args);
+#if UNITY_IOS
+        return Paths.ReplacePlatformPathText(string.Format(format, args));
```

Comment:
While most callers of `GetStringAndFormat` indeed deal with paths, I would advice against internalizing a `ReplacePlatformPathText` call as it becomes a "surprise feature".
How about this: we leave `GetStringAndFormat` unchanged, and add another method:
```
public static string GetStringAndFormatIncludingPaths(string formatKey, params object[] args) {
    return Paths.ReplacePlatformPathText(GetStringAndFormat(formatKey, args));
}
```
Then we replace `GetStringAndFormat` calls with the new method where necessary. I see 2 calls that don't need to be replaced: the one in `Game.UpdateFever` and the one in `TouchscreenTestPanel.Update`.